src/model/SyntaxNodeType.ts

```diff
@@ -68,6 +77,13 @@ export enum SyntaxNodeType {
     Parameters = "parameters",
     FunctionParameter = "function_parameter",
     FunctionParameterMode = "function_parameter_mode",
+    // arithmetic operators
+    Add = "+",
+    Subtract = "-",
+    Multiply = "*",
+    Divide = "/",
+    Modulus = "%",
+    EqualsSign = "=",
```

Comment:
You forgot the fun ones: `eq`, `ne`, `lt` and etc. We need to support those too.

Reply:
There are also compound operators `+=` , `-=`, `/=`, `*=`  and the safe navigation operator `?:`

Reply:
Yeah, that's right, we don't need those here. But regarding `?:` @gustason could you create a ticket for it, so we won't forget. I think on tree-sitter side it should be easy to implement.